test/ngResource/resourceSpec.js

```diff
@@ -1052,7 +1052,8 @@ describe("basic usage", function() {
     it('should call the error callback if provided on non 2xx response', function() {
       $httpBackend.expect('GET', '/CreditCard/123').respond(ERROR_CODE, ERROR_RESPONSE);
 
-      CreditCard.get({id:123}, callback, errorCB);
+      var ccs = CreditCard.get({id:123}, callback, errorCB);
```

Comment:
Yes, I think an alternative that doesn't report an error when an error-callback is used is better.
lgalfaso@9f7ed38 isn't ideal either, as it's a BC :(